docs/RESOURCES.md

```diff
@@ -10,6 +10,7 @@
 * [StackOverflow ](http://stackoverflow.com/questions/tagged/c%23) can be used to search for your problem and see if it has been answered already. You can also ask and answer questions.
 * [/r/csharp](https://www.reddit.com/r/csharp) is the C# subreddit.
 * [@dotnet](https://twitter.com/DotNet) is the official .NET Twitter account.
+* [Gitter](https://gitter.im/exercism/xcsharp) is the C# Gitter room.
```

Comment:
Perhaps we can reword this slightly, to make it more clear that people can ask questions about exercism's C# track here. Maybe something like: 

"For questions about the Exercism C# track, please use the [Gitter room](https://gitter.im/exercism/xcsharp)."

Or:

"To get support for the C# track, please use the [Gitter room](https://gitter.im/exercism/xcsharp)."

Or perhaps something else entirely. What do you feel is the best option?

The main point is that we should explain why people should go to the Gitter room. Sorry I didn't make that clear in the issue.